hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdb/util/TestManagedSstFileReader.java

```diff
@@ -91,19 +91,24 @@ private Map<String, Integer> createKeys(int startRange, int endRange) {
             i -> i % 2));
   }
 
-  private Pair<Map<String, Integer>, List<String>> createDummyData(
+  private Pair<TreeMap<String, Integer>, List<String>> createDummyData(
```

Comment:
What's the reason behind changing the type here? It opposes to interface over implementation.

Reply:
Changed it to SortedMap, getTestingBounds function needs SortedMap as an argument.